packages/phone-number-privacy/signer/src/pnp/endpoints/sign/action.ts

```diff
@@ -60,14 +60,20 @@ export class PnpSignAction implements Action<SignMessageRequest> {
             // We fail open and service requests on full-node errors to not block the user.
             // Error messages are stored in the session and included along with the signature in the response.
             quotaStatus.totalQuota = Number.MAX_SAFE_INTEGER
-            session.logger.error(ErrorMessage.FAILING_OPEN)
+            session.logger.error(
```

Comment:
why not `session.logger.warn(...`? (may also make sense to add auditing our logging levels to the log auditing ticket, since I guess we've been not as precise as we maybe should have been?)